docs/source/upgrade-guide/index.md

```diff
@@ -30,6 +30,94 @@ the latest changes, and propose to you to merge the changes, so you can run it o
 
 ## Upgrading to Volto 16.x.x
 
+### `volto-slate` is now core
+
+From Volto 16.0.0-alpha.15 onwards, `volto-slate` is integrated into Volto core and enabled as the default text block.
+The previous text block `text` based on `draftJS` is now deprecated and is restricted (hidden) in Volto bootstrap.
+This is a major change and should be planned in advance before you install 16.0.0-alpha.15 or later.
+
+```{versionadded} 16.0.0-alpha.15
+`volto-slate` added to Volto core as the default text block.
+```
+
+```{deprecated} 16.0.0-alpha.15
+`text` text block based on `draftJS` is now deprecated and will be removed from core in Volto 18.
+```
+
+```{note}
+Volto 16 is intended to be the long-term support (LTS) version used when Plone 6 Final is released.
+Volto 16 supports `draftJS` text block deployments but please notice it's deprecated from this version on.
+As such you will have the choice to continue using `draftJS` text block in your projects but developers and integrators are strongly encouraged to migrate existing sites using `draftJS` text blocks to migrate to `Slate` text blocks.
```

Comment:
One minor detail. When we use "Slate" that means a proper noun or the product named Slate. When we use "`slate`", that is the text block. I missed this distinction on my first pass.

```suggestion
As such, you are strongly encouraged to migrate existing sites that use `draftJS` text blocks to `slate` text blocks.
```

Reply:
Resolved in https://github.com/plone/volto/pull/3529